CONTRIBUTING.md

```diff
@@ -270,3 +234,28 @@ See [`src/test/regress/README.md`](https://github.com/citusdata/citus/blob/maste
 User-facing documentation is published on [docs.citusdata.com](https://docs.citusdata.com/). When adding a new feature, function, or setting, you can open a pull request or issue against the [Citus docs repo](https://github.com/citusdata/citus_docs/).
 
 Detailed descriptions of the implementation for Citus developers are provided in the [Citus Technical Documentation](src/backend/distributed/README.md). It is currently a single file for ease of searching. Please update the documentation if you make any changes that affect the design or add major new features.
+
+# Making a pull request ready for reviews
+
+Asking for help and asking for reviews are two different things. When you're asking for help, you're asking for someone to help you with something that you're not expected to know.
+
+But when you're asking for a review, you're asking for someone to review your work and provide feedback. So, when you're asking for a review, you're expected to make sure that:
+
+* Your changes don't perform **unnecessary line addition / deletions / style changes on unrelated files / lines**.
+
+* All CI jobs are **passing**, including **style checks** and **flaky test detection jobs**.
```

Comment:
Would be good to mention here that if you're an outside contributor, it's fine to submit before this is the case. Because it's hard for outside contributors to run CI themselves.

Reply:
Maybe just in general at the start of this section. Something like:

> If you're a new/outside contributor the following doesn't necessarily apply to you. We always welcome PRs from users even if they don't conform to these things.